<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>790</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1214</width>
    <height>790</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1214</width>
    <height>790</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Defect Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/logo.ico</normaloff>:/logo.ico</iconset>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="mainWidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>741</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTabWidget" name="widgetDisplay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1011</width>
       <height>731</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="widgetTab1">
      <attribute name="title">
       <string>Current Layer</string>
      </attribute>
      <widget class="QLabel" name="layerDisplay">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>991</width>
         <height>681</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widgetTab2">
      <attribute name="title">
       <string>Coat Explorer</string>
      </attribute>
      <widget class="QSlider" name="sliderLayerSelectCoat">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>10</y>
         <width>22</width>
         <height>681</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widgetTab3">
      <attribute name="title">
       <string>Scan Explorer</string>
      </attribute>
      <widget class="QSlider" name="sliderLayerSelectScan">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>10</y>
         <width>22</width>
         <height>681</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widgetTab4">
      <attribute name="title">
       <string>Meltpool Analysis</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupDisplayOptions">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>25</y>
       <width>170</width>
       <height>200</height>
      </rect>
     </property>
     <property name="title">
      <string>Display Options</string>
     </property>
     <widget class="QProgressBar" name="progressImageProcessing">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>16</width>
        <height>127</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkToggleOverlay">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>110</width>
        <height>20</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Toggle Overlay</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layoutDisplayOptions">
       <item>
        <widget class="QRadioButton" name="radioRaw">
         <property name="text">
          <string>Raw Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioCorrection">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Correction</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioCrop">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Crop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioCLAHE">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Equalization</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonStop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1085</x>
       <y>390</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>170</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>21</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>â–ˆ</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonStart">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>289</y>
       <width>110</width>
       <height>50</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonPause">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1085</x>
       <y>350</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string> â–Œâ–Œ</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonSet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>485</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SET</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineLayer">
     <property name="geometry">
      <rect>
       <x>1042</x>
       <y>486</y>
       <width>51</width>
       <height>29</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>LAYER</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLayerNumber">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>450</y>
       <width>71</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupPositionAdjustment">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>530</y>
       <width>150</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>Position Adjustment</string>
     </property>
     <widget class="QPushButton" name="buttonCrop">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="toolTip">
       <string>Crop toggle</string>
      </property>
      <property name="text">
       <string>â•¬</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonShiftLeft">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>â—„</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonShiftRight">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>â–º</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonShiftDown">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>â–¼</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonShiftUp">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>â–²</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRotateACW">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>â†“</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonRotateCW">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>â†“</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineShiftPixels">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineShiftDegrees">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>22</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.05</string>
      </property>
      <property name="maxLength">
       <number>5</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelDegrees">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>22</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Degrees</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPixels">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>50</y>
        <width>50</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pixels</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonBoundary">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="toolTip">
       <string>Move overlay boundary</string>
      </property>
      <property name="text">
       <string>â–¡</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonPhase">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1138</x>
       <y>485</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>COAT</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelLayerNumberTitle">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>430</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Layer Number&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonInitialize">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>240</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>10</red>
           <green>167</green>
           <blue>170</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>10</red>
           <green>167</green>
           <blue>170</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>INITIALIZE</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewBuild"/>
    <addaction name="actionOpenBuild"/>
    <addaction name="separator"/>
    <addaction name="actionExportImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <widget class="QMenu" name="menuPartFiles">
     <property name="title">
      <string>Part Files</string>
     </property>
     <addaction name="actionCLS"/>
     <addaction name="actionCLI"/>
    </widget>
    <widget class="QMenu" name="menuReportSettings">
     <property name="title">
      <string>Report Settings</string>
     </property>
     <addaction name="actionDefectActions"/>
     <addaction name="actionNotificationSetup"/>
    </widget>
    <widget class="QMenu" name="menuImageCalibration">
     <property name="title">
      <string>Image Calibration</string>
     </property>
     <addaction name="actionAdjustCrop"/>
     <addaction name="actionAdjustRotation"/>
    </widget>
    <addaction name="actionBuildPlatform"/>
    <addaction name="menuPartFiles"/>
    <addaction name="separator"/>
    <addaction name="menuImageCalibration"/>
    <addaction name="separator"/>
    <addaction name="menuReportSettings"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionStartBuild"/>
    <addaction name="actionPauseBuild"/>
    <addaction name="separator"/>
    <addaction name="actionStopBuild"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetup"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <action name="actionBuildPlatform">
   <property name="text">
    <string>Build Platform</string>
   </property>
  </action>
  <action name="actionCLS">
   <property name="text">
    <string>CLS</string>
   </property>
  </action>
  <action name="actionCLI">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>CLI</string>
   </property>
  </action>
  <action name="actionNewBuild">
   <property name="text">
    <string>New Build...</string>
   </property>
  </action>
  <action name="actionOpenBuild">
   <property name="text">
    <string>Open Build...</string>
   </property>
  </action>
  <action name="actionExportImage">
   <property name="text">
    <string>Export Image...</string>
   </property>
  </action>
  <action name="actionError_actions">
   <property name="text">
    <string>Defect actions</string>
   </property>
  </action>
  <action name="actionDefectActions">
   <property name="text">
    <string>Defect Actions</string>
   </property>
  </action>
  <action name="actionNotificationSetup">
   <property name="text">
    <string>Notification Setup</string>
   </property>
  </action>
  <action name="actionAdjustCrop">
   <property name="text">
    <string>Adjust Crop</string>
   </property>
  </action>
  <action name="actionAdjustRotation">
   <property name="text">
    <string>Adjust Rotation</string>
   </property>
  </action>
  <action name="actionStartBuild">
   <property name="text">
    <string>Start Build</string>
   </property>
  </action>
  <action name="actionPauseBuild">
   <property name="text">
    <string>Pause Build</string>
   </property>
  </action>
  <action name="actionStopBuild">
   <property name="text">
    <string>Stop Build</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
